src/utilities/gql.ts

```diff
@@ -890,7 +890,7 @@ const gql = {
           dataset {
             profiles {
               name
-              profile_segments(order_by: { start_offset: asc }) {
+              profile_segments(where: {is_gap: {_eq: false}}, order_by: { start_offset: asc }) {
```

Comment:
Just a small formatting update. Can we add spaces to stay consistent with the other queries in this file:

```
where: { is_gap: { _eq: false } }
```

Reply:
done